test/unit/url.test.js

```diff
@@ -526,7 +526,7 @@ describe('Url', function () {
             expect(jsonified.protocol).to.eql(rawUrl.protocol);
             expect(jsonified.host).to.eql(rawUrl.host.split('.'));
             expect(jsonified.port).to.eql(rawUrl.port);
-            expect(jsonified.path).to.eql(rawUrl.path.split('/'));
+            expect(jsonified.path).to.eql(rawUrl.path.split('/').slice(1));
```

Comment:
Earlier, all paths would be of the form `['', 'first', 'second']`, since split would interpret the leading slash as an empty string at the 0th index in the path array. Now that has been trimmed away, the expected comparison should not expect an empty string as the first segment in the path.

Reply:
Note that the first empty string allows us to distinguish between `/abc` and `abc`. Is that going to break now?

Reply:
@kunagpal - that’s a good question from Aniket. Check and let us know. 

Also, the tests need to not use these long haul fixtures - maybe a separate work later to remove all fixtures

Reply:
Sorry, this had slipped through. Now that you've mentioned it, this is effectively a breaking change. Where do we need to distinguish between `/abc` and `abc`?

Reply:
I can tell you where this logic is used. Any time you're converting the path segment array to a string. So.. in the URL stringification logic, collection transformation, a few places in the documenter, probably in the app too. So it _could_ be the difference between `blah.com:8080/abc` and `blah.com:8080abc`

We need to distinguish between the two because it's not the SDK's or the collection format's job to guess what the user has entered in the URL bar, or try to correct it.